client/batch.go

```diff
@@ -46,7 +45,7 @@ type Batch struct {
 	//   // string(b.Results[0].Rows[0].Key) == "a"
 	//   // string(b.Results[1].Rows[0].Key) == "b"
 	Results    []Result
```

Comment:
It'll probably still wind up having some users who can appreciate it (sub-sql maintenance/inspection tools, etc).